src/peer-id.js

```diff
@@ -20,15 +20,17 @@ async function downloadPeerIdFile(dispatcher) {
 }
 
 async function getPeerId(dispatcher = defaultDispatcher) {
+  const { createFromJSON, createEd25519PeerId } = await loadEsmModule('@libp2p/peer-id-factory')
```

Comment:
I'm not sure this is right, I took from examples and `libp2p` code

Reply:
I would say so